tests/test_models.py

```diff
@@ -0,0 +1,11 @@
+from dolib.models import Network, Region
+
+
+def test_repr() -> None:
+    network = Network(
+        ip_address="1.1.1.1", netmask="255.255.255.0", gateway="1.1.1.254", type="test"
+    )
+    assert network.__repr__() == "Network(1.1.1.1/255.255.255.0 [test])"
```

Comment:
```suggestion
    assert repr(network) == "Network(1.1.1.1/255.255.255.0 [test])"
```

`repr()` could be used here instead of the dunder method